myapp/api/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.urls import path
 from rest_framework.routers import DefaultRouter
-from . import views
+from .views import views
```

Comment:
could you change this to myapp.api.views? I think its better for consistency in how we do imports